model_analyzer/config/run/model_run_config.py

```diff
@@ -99,9 +99,9 @@ def representation(self):
 
         return self.perf_config().representation()
 
-    def is_legal_combination(self):
+    def check_for_client_vs_model_batch_size(self):
```

Comment:
These are private methods and should have an underscore in front of the method name

Reply:
Yes, done the modification 👍